.github/workflows/CPL_latest-rc-latest.yaml

```diff
@@ -0,0 +1,15 @@
+name: Check CPL latest/rc/latest
+
+on:
+  schedule:
+    - cron: "20 10 * * 1-5"
```

Comment:
Do you have any preference on when and/or how frequent this action should run?

Reply:
I can set it to match the others for the RC branch test matrix - they usually run every day at midnight. 

If there is a way to easily disable outside of feature freeze without updating the PR before and after each one, I don't know of it. The other RC branch tests just run every day no matter whether its feature freeze or not 🤷‍♀️

Reply:
> The other RC branch tests just run every day no matter whether its feature freeze or not 🤷‍♀️

Actions can be enabled/disabled pretty easily from the actions tab :)
![image](https://github.com/PennyLaneAI/catalyst/assets/29467667/a40e8edd-80fb-4c13-89eb-8c579f42ca58)

Reply:
Ah, good to know, thanks!